falco/values.yaml

```diff
@@ -133,9 +133,26 @@ falco:
     - /etc/falco/falco_rules.yaml
     - /etc/falco/falco_rules.local.yaml
     - /etc/falco/k8s_audit_rules.yaml
+    # - /etc/falco/aws_cloudtrail_rules.yaml
     - /etc/falco/rules.d
   # - /etc/falco/rules.optional.d
 
+  plugins:
+    - name: cloudtrail
+      library_path: libcloudtrail.so
+      init_config: ""
+      open_params: ""
+    - name: json
+      library_path: libjson.so
+      init_config: ""
+
+  # Setting this list to empty ensures that the above plugins are *not*
+  # loaded and enabled by default. If you want to use the above plugins,
+  # set a meaningful init_config/open_params for the cloudtrail plugin
+  # and then change this to:
+  # load_plugins: [cloudtrail, json]
+  loadPlugins: []
```

Comment:
Here what I've found in current version of documentation:
```
The plugins property in falco.yaml defines the set of plugins that can be loaded by Falco. The property contains a list of plugin definitions. Each plugin definition has:

    name: The name of the plugin.
    library_path: The path to the shared library of the plugin.
    init_config: A configuration string (typically JSON) that is passed to the plugin when the plugin is initialized.
    open_params: A configuration string that is passed to the plugin when the plugin open()s a new stream of events.

By default, every plugin in the plugins list will be loaded. If present, the load_plugins property contains specific name entries from plugins that are loaded instead
```
What's the real behavior?

Source: https://falco.org/blog/falco-plugins-early-access/#configuring-plugins-in-falco

Reply:
The real behavior is 
```
Setting this list to empty ensures that the above plugins are *not* loaded and enabled by default.
```

The documentation needs to be improved because it's misleading.